tools/build_paks.py

```diff
@@ -13,30 +13,104 @@
+        "-unversioned",                          # save all of the cooked packages without versions
+        "-stdout",                               # ensure log output is written to the terminal 
+        "-fullstdoutlogoutput",                  # ensure log output is written to the terminal
+        "-nologtimes"]                           # don't print timestamps next to log messages twice
+    spear.log(f"    Executing: {' '.join(cmd)}")
+    subprocess.run(cmd, check=True)
+
+    # create the output_dir
+    os.makedirs(output_dir, exist_ok=True)
+
+    # create manifest file in output dir
+    with open(txt_file, mode="w") as f:
+        for cooked_include_dir in cooked_include_dirs:
+            for cooked_include_file in glob.glob(os.path.realpath(os.path.join(cooked_include_dir, "**", "*.*")), recursive=True):
+
+                cooked_include_file_posix = cooked_include_file.replace(ntpath.sep, posixpath.sep)
+                asset_name = cooked_include_file_posix.removeprefix(unreal_project_cooked_dir_posix + posixpath.sep)
+
+                if asset_name not in default_pak_asset_names:
+                    assert cooked_include_file_posix.startswith(unreal_project_cooked_dir_posix)
+                    cooked_mount_file_posix = posixpath.join("..", "..", "..", cooked_include_file_posix.replace(unreal_project_cooked_dir_posix + posixpath.sep, ""))
+                    f.write(f'"{cooked_include_file_posix}" "{cooked_mount_file_posix}" "" \n')
+
+    # build pak file
+    cmd = [unreal_pak_bin, pak_file, "-create=" + txt_file, "-platform=" + platform, "-multiprocess", "-compressed"]
+    spear.log(f"    Executing: {' '.join(cmd)}")
+    subprocess.run(cmd, check=True)
+
+    assert os.path.exists(pak_file)
+    spear.log(f"    Successfully built: {pak_file}")
+
+    # remove symlinks
+    for symlink_dir_physical, symlink_dir_virtual in symlink_dirs.items():
+        spear.log(f"    Removing symlink: {symlink_dir_virtual}")
+        spear.remove_path(symlink_dir_virtual)
+
+
 if __name__ == '__main__':
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--unreal_engine_dir", required=True)
-    parser.add_argument("--unreal_project_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim")))
     parser.add_argument("--output_dir", required=True)
     parser.add_argument("--version_tag", required=True)
-    parser.add_argument("--skip_create_symlinks", action="store_true")
-    parser.add_argument("--perforce_content_dir")
+    parser.add_argument("--perforce_content_dir", required=True)
+    parser.add_argument("--unreal_project_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim")))
+    parser.add_argument("--spearsim_executable_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "build", "SpearSim-Win64-Shipping")))
```

Comment:
Let's call this `build_dir` because that is what it's called in the `build_executable` tool, and then let's set the full path in our platform-specific logic.

Reply:
Aah, right. Missed the `Win64` in the path.